index.bs

```diff
@@ -2909,7 +2909,7 @@ Determining the W3C Decision</h4>
 	<em class=rfc2119>must not</em> include any increase to the scope.
 	If any such change is desired,
 	it <em class=rfc2119>must</em> be returned for additional work
-	and go through a complete new [=AC Review=].
+	and go through a completely new [=AC Review=].
```

Comment:
I would not take this change. "complete" insists that the AC Review must be a full, normal one, as opposed to a partial one. The suggested rephrasing instead intensifies "new", which I don't think is necessary.